intr.S

```diff
@@ -0,0 +1,91 @@
+    .set noreorder // Disable automatic instruction reordering
+
+    /* Interrupts vector */
+    .section .exception
+    .globl _interrupt_handler
+    .org 0x200
+_interrupt_handler:
```

Comment:
Have you compared your CPU context save / restore routines with [exception.S](http://nxr.netbsd.org/xref/src-freebsd/sys/mips/mips/exception.S#268)?